### PR TITLE
edge case and fixed nav

### PR DESCRIPTION
**Pull request by me-liu**
### Description

[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Fixed navigation so that the buttons actually direct to different pages, removed Pressable.
Added an edge case for StyledButton if there is no file attached. 

### Screenshots
image

[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review

[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

The last commit

## Next steps

[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
Should be finished

### Online sources

[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @stephaniewong2

### CODE REVIEW

File: src/Components/StyledButton.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Link } from 'expo-router';
```
Comment (stephaniewong2) [inline src/Components/StyledButton.tsx]: Let's put the StyledButton and its styles sheet into a folder inside `Components`!

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]
Everything else looks good to me!
